runelite-client/src/main/java/net/runelite/client/plugins/chatcommands/ChatCommandsPlugin.java

```diff
@@ -623,7 +623,7 @@ else if (result.equals("were defeated"))
 		matcher = GUARDIANS_OF_THE_RIFT_PATTERN.matcher(message);
 		if (matcher.find())
 		{
-			int kc = Integer.parseInt(matcher.group(1));
+			int kc = Integer.parseInt(matcher.group(1).replaceAll(",", ""));
```

Comment:
```suggestion
			int kc = Integer.parseInt(matcher.group(1).replace(",", ""));
```
to avoid regex overhead

Reply:
The Huntor Rumour, Hollowed Sep and Grand Hallowed Coffin both use replaceAll, I just kept on the tradition. 

I do not think anybody is going to get 1,000,000 in any of these 3 fwiw. Just copied what others were doing.

Reply:
re: multiple commas - `String#replace(String, String)` doesn't stop at the first occurrence